server/index.js

```diff
@@ -79,6 +79,34 @@ app.get('/api/cart', (req, res, next) => {
   }
 });
 
+app.post('/api/orders/', (req, res, next) => {
+  if (!req.session.cartId) {
+    throw new ClientError('Order does not exist', 400);
+  } else if (req.body.name && req.body.creditCard && req.body.shippingAddress) {
+    const orderEntry = `
+      insert into "orders" ("cartId", "name", "creditCard", "shippingAddress")
+      values ($1, $2, $3, $4)`;
```

Comment:
You can just add a `returning` clause here so you don't have to query the database again.